src/ui/starter-select-ui-handler.ts

```diff
@@ -2415,7 +2415,7 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
       }
     });
 
-    this.starterSelectScrollBar.setPages(Math.ceil((this.filteredStarterContainers.length - 81) / 9) + 1);
+    this.starterSelectScrollBar.setPages(Math.ceil(this.filteredStarterContainers.length / 9));
```

Comment:
```suggestion
    this.starterSelectScrollBar.setPages(Math.max(Math.ceil(this.filteredStarterContainers.length / 9)), 1);
```

Shouldn't we ensure that the minimum value is set to 1 to prevent division by 0 issues in setPages?